### DISCUSSION TITLE
I have an existing jupyter-book project created in v0.12.2. How do I upgrade the theme?

### DISCUSSION THREAD

**Discussion by simkimsia** [executablebooks/meta#725] [Q&A]
I just upgraded the jupyter-book program from v0.12.2 to v0.12.3

<img width="1099" alt="image" src="https://user-images.githubusercontent.com/245021/163348954-5c95edfa-cc90-461c-89f3-63ec112c19f9.png">

I built a jupyter-book project called `lessons-learned-making-mrr` using v0.12.2.

I know the sphinx-book-theme, a dependency for jupyter-book, has been bumped to v0.3 https://github.com/executablebooks/jupyter-book/commit/0882d8ebccb45bd8d9e725566d7612f050e47c07 in the main branch.

I think v0.12.3 does not have that change yet.

My question is when the sphinx-book-theme is part of a released pypi version, how do I upgrade the theme in my book `lessons-learned-making-mrr` project?

Or there is no need to because each time, i run `jupyter-book build` command it will instantly uses the installed `sphinx-book-theme` to rebuild the html pages for it?

**Comment by choldgraf** [comment]
To update the theme, run the following from the environment where jupyter-book is installed:

```
pip install -U sphinx-book-theme
```

**Reply by simkimsia** [reply]
this is what happens when I did what you said

![image](https://user-images.githubusercontent.com/245021/164878253-6bb25b03-135c-435f-8683-5e25ad77117b.png)

As mentioned previously,

> I think v0.12.3 does not have that change yet.

**Reply by choldgraf** [reply]
Ah - correct `pip` will not let you install an incompatible version. This version has been bumped on the `master` branch. Can you try running jupyter book from master?

```console
$ pip install git+https://github.com/executablebooks/jupyter-book
```

should work. Though we'll try to cut a release soon that makes the book theme work more easily
